content/log/processing.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Processing
+kind: Documentation
+autotocdepth: 2
+hideguides: true
+customnav: lognav
+---
+
+<div class="alert alert-info">
+Datadog's log management solution is is currently in private beta. If you would like to apply to it, please fill out <a href="https://www.datadoghq.com/log-management/">this form</a>.
+</div>
+
+## Overview
+
+To access the processing panel please use the upper left menu:
+{{< img src="log/processing/pipelines_panel.png" alt="Pipelines pannel" >}}
+
+Your processing view should look like this:
+{{< img src="log/processing/pipelines_view.png" alt="Pipelines view" >}}
+
+## Processing Pipelines 
+### Pipelines Goal 
+
+**A processing pipeline applies over a filtered subset of incoming logs a list of sequential processors.**
+
+With pipelines you have the opportunity to parse & enrich your logs through a sequential chains of [processors](/log/processing/#processors) in order to extract meaningful information or to extract attributes from semi-structured text to re-use them as [facet](/log/explore/#facets).
+
+Each log that comes through the pipelines is tested against every pipeline filter. If it matches one then all the [processors](#processors) are applied sequentially before moving to the next pipeline.
+
+So for instance a processing pipeline can transform this log:
```

Comment:
As discussed:
Let's use a java example for the pipeline:

2017-09-21T10:38:21.004 Error Connection refused for user 12345

Then we parse it with attribute `date` , `level`, `error_msg`

In the sketch we use the same example and we show the metadata so user can understand that at the begining log has severity "info" and a log date matching the API reception time.
Then with the remapper we do not have a "severity" and "timestamp" attribute but the main log severity and date are updated.